integration/groupby_query_test.go

```diff
@@ -0,0 +1,85 @@
+// +build integration
+
+package integration
+
+import (
+	"encoding/json"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestGroupByQuery(t *testing.T) {
```

Comment:
`TestGroupByQueryNoFilterWithCalculationsNoOrder`?

Reply:
See comment above on adding diff types of queries in the tests.